shells/electron/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "main": "./index.js",
+  "scripts": {
+    "build": "webpack --config webpack.backend.js && webpack --config webpack.config.js"
```

Comment:
I think we can re-use `webpack` that was installed via root `react-devtools/package.json`